starters/gatsby-starter-minimal-ts/gatsby-config.js

```diff
@@ -1,7 +1,7 @@
 /** @type {import('gatsby).GatsbyConfig} */
 module.exports = {
   siteMetadata: {
-    siteUrl: `https://www.yourdomain.tld`,
+      siteUrl: `https://www.yourdomain.tld`,
```

Comment:
The rest of this appears to be indented with 2 spaces, but not `siteUrl` is indented with 4. I think we should go with one or the other for the whole file.

Reply:
I've edited it for now since our CLI adds `title` with 4 indendations:

```
test-03 on  main is 📦 v1.0.0 via ⬢ v16.13.0
➜ cat gatsby-config.js
/** @type {import('gatsby).GatsbyConfig} */
module.exports = {
  siteMetadata: {
      title: ``,
    siteUrl: `https://www.yourdomain.tld`
  },
  plugins: ["gatsby-plugin-vanilla-extract"]
};%
```

And it's how we have it in the other starter: https://github.com/gatsbyjs/gatsby/blob/master/starters/gatsby-starter-minimal/gatsby-config.js

So that's why I did it this way